scripts/overview.json

```diff
@@ -2943,6 +2974,7 @@
           "dashLength": 10,
           "dashes": false,
           "datasource": "${DS_TEST-CLUSTER}",
+          "description": "the number of transaction related conflicts",
```

Comment:
The description is not accurate. Lock resolving doesn't mean conflicts. If you must add description to it, I prefer just use its literal meaning.